WooCommerce/WooCommerceUITests/Tests/ReviewsTests.swift

```diff
@@ -0,0 +1,31 @@
+import UITestsFoundation
+import XCTest
+
+final class ReviewsTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        continueAfterFailure = false
+
+        let app = XCUIApplication()
+        app.launchArguments = ["logout-at-launch", "disable-animations", "mocked-wpcom-api", "-ui_testing"]
+        app.launch()
+        try LoginFlow.logInWithWPcom()
+    }
+
+    func testReviewsScreenLoad() throws {
+        let reviews = try GetMocks.readReviewsData()
+
+        // Extra step needed to get products mock data to appear on Reviews screen
+        try TabNavComponent()
+            .gotoProductsScreen()
```

Comment:
There might be a better way to do this, for now, the test needs to land on the products screen to get product information to be displayed on the reviews screen. Without this step, the product will show as "unknown" on the reviews screen.

Reply:
Since it is a required step but not directly tied to the test, it could make sense to move it into `setUpWithError()` and maybe even the `TabNavComponent().gotoReviewsScreen()` as the tests in this file are likely to start from the ReviewsScreen.

Regarding the need to visit the Products page first, is it possible that the reviews could be part of a previous API call to another endpoint?

Reply:
Good point! Yeah, those navigation steps should be needed for all tests on this screen. I'll look into moving that to `setUpWithError()`. The need for the extra step could be a missing mock call on the Reviews screen, will need to investigate further (first need to figure out how to manually get that screen populated with reviews before testing the calls on Charles). 

Ideally, I'd like to remove the extra step for the test to go to the Products screen first.